scss/_settings.scss

```diff
@@ -189,6 +189,11 @@ $fd-forms-select-background-image--active: "data:image/svg+xml;base64,PHN2ZyB4bW
 
+$fd-status-indicator-available: "data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMiAxMiI+PGRlZnM+PHN0eWxlPi5jbHMtMXtmaWxsOiMxMzg4NDM7fS5jbHMtMntmaWxsOiNmZmY7fTwvc3R5bGU+PC9kZWZzPjx0aXRsZT5BcnRib2FyZCAxPC90aXRsZT48ZyBpZD0iSWNvbkJ1c3kiPjxjaXJjbGUgaWQ9IldoaXRlQ2lyY2xlX0xhcmdlIiBkYXRhLW5hbWU9IldoaXRlQ2lyY2xlIExhcmdlIiBjbGFzcz0iY2xzLTEiIGN4PSI2IiBjeT0iNiIgcj0iNiIvPjwvZz48ZyBpZD0iT25saW5lLVN0YXR1c19BdmFpbGFibGUtQ29weS0yIiBkYXRhLW5hbWU9Ik9ubGluZS1TdGF0dXMvQXZhaWxhYmxlLUNvcHktMiI+PGcgaWQ9Ikljb25BdmFpbGFibGUiPjxwYXRoIGlkPSJDaGVja01hcmsiIGNsYXNzPSJjbHMtMiIgZD0iTTUuNzMsNi4zMSw3LjExLDMuNTVhMSwxLDAsMCwxLDEuNzguOWwtMiw0YTEsMSwwLDAsMS0xLjYuMjZsLTItMkExLDEsMCwwLDEsNC43MSw1LjI5WiIvPjwvZz48L2c+PC9zdmc+" !default;
+$fd-status-indicator-away:"data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMiAxMiI+PGRlZnM+PHN0eWxlPi5jbHMtMXtmaWxsOiNlODc0MjQ7fS5jbHMtMntmaWxsOiNmZmY7fTwvc3R5bGU+PC9kZWZzPjx0aXRsZT5BcnRib2FyZCAxPC90aXRsZT48ZyBpZD0iSWNvbkJ1c3kiPjxjaXJjbGUgaWQ9IldoaXRlQ2lyY2xlX0xhcmdlIiBkYXRhLW5hbWU9IldoaXRlQ2lyY2xlIExhcmdlIiBjbGFzcz0iY2xzLTEiIGN4PSI2IiBjeT0iNiIgcj0iNiIvPjwvZz48ZyBpZD0iU3ltYm9scyI+PGcgaWQ9Ik9ubGluZS1TdGF0dXNfQXdheS1Db3B5LTIiIGRhdGEtbmFtZT0iT25saW5lLVN0YXR1cy9Bd2F5LUNvcHktMiI+PGcgaWQ9Ikljb25Bd2F5Ij48cGF0aCBpZD0iQ2xvY2tIYW5kcyIgY2xhc3M9ImNscy0yIiBkPSJNNi4wNSw3YTEsMSwwLDAsMS0xLTFWM2ExLDEsMCwwLDEsMiwwVjVoMmExLDEsMCwwLDEsMCwyWiIvPjwvZz48L2c+PC9nPjwvc3ZnPg==" !default;
+$fd-status-indicator-busy:"data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMiAxMiI+PGRlZnM+PHN0eWxlPi5jbHMtMXtmaWxsOiNlZDFjMjQ7fS5jbHMtMntmaWxsOiNmZmY7fTwvc3R5bGU+PC9kZWZzPjx0aXRsZT5BcnRib2FyZCAxPC90aXRsZT48ZyBpZD0iSWNvbkJ1c3kiPjxjaXJjbGUgaWQ9IldoaXRlQ2lyY2xlX0xhcmdlIiBkYXRhLW5hbWU9IldoaXRlQ2lyY2xlIExhcmdlIiBjbGFzcz0iY2xzLTEiIGN4PSI2IiBjeT0iNiIgcj0iNiIvPjwvZz48ZyBpZD0iU3ltYm9scyI+PGcgaWQ9Ik9ubGluZS1TdGF0dXNfQnVzeS1Db3B5LTIiIGRhdGEtbmFtZT0iT25saW5lLVN0YXR1cy9CdXN5LUNvcHktMiI+PGcgaWQ9Ikljb25CdXN5LTIiIGRhdGEtbmFtZT0iSWNvbkJ1c3kiPjxwYXRoIGlkPSJXaGl0ZS1DaXJjbGVfU21hbGwiIGRhdGEtbmFtZT0iV2hpdGUtQ2lyY2xlIFNtYWxsIiBjbGFzcz0iY2xzLTIiIGQ9Ik04LDZBMiwyLDAsMSwxLDYsNCwyLDIsMCwwLDEsOCw2Ii8+PC9nPjwvZz48L2c+PC9zdmc+" !default;
+$fd-status-indicator-offline:"data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMiAxMiI+PGRlZnM+PHN0eWxlPi5jbHMtMXtmaWxsOm5vbmU7c3Ryb2tlOiM1MTU1NTk7c3Ryb2tlLXdpZHRoOjJweDt9PC9zdHlsZT48L2RlZnM+PHRpdGxlPkFydGJvYXJkIDE8L3RpdGxlPjxnIGlkPSJTeW1ib2xzIj48ZyBpZD0iT25saW5lLVN0YXR1c19PZmZsaW5lLUNvcHkiIGRhdGEtbmFtZT0iT25saW5lLVN0YXR1cy9PZmZsaW5lLUNvcHkiPjxjaXJjbGUgaWQ9Ik92YWwiIGNsYXNzPSJjbHMtMSIgY3g9IjYiIGN5PSI2IiByPSI1Ii8+PC9nPjwvZz48L3N2Zz4=" !default;
+
```

Comment:
Hey @saadmhybris ~ we'll want to pull out the colors that are baked into the SVG URL's and reference the status colors in our library. For these guys specifically, it can be accomplished by just setting the background color of a box with a border-radius and the SVG URL's end up just being the shapes in the foreground. 

If you need a hand generating those URLs let me know since i've got illustrator installed. :)